RLTest/env.py

```diff
@@ -114,13 +114,14 @@ class Env:
     defaultUseAof = None
     defaultDebugger = None
     defaultExitOnFailure = False
+    defaultDecodeResponses = True if (sys.version >= '3') else False
 
     RTestInstance = None
 
     defaultDebugPrints = False
     defaultNoCatch = False
 
-    EnvCompareParams = ['module', 'moduleArgs', 'env', 'useSlaves', 'shardsCount', 'useAof']
+    EnvCompareParams = ['module', 'moduleArgs', 'env', 'useSlaves', 'shardsCount', 'useAof', 'decode_responses']
```

Comment:
This list contains parameters that might be override by a specific test, decode_responses is not such a parameter.

Reply:
Some tests may actually expect non-UTF-8 results - such tests cannot use the default decode() and have to contain special handling.  There are tests like this in the ReJSON module, for example, as it needs to verify handling of wide-character strings.  So, I think it must be an optional parameter.

Reply:
In this case we also need to allow to override it in the command line execution...

Reply:
@MeirShpilraien yep - I'll update the PR